pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2445,6 +2512,21 @@
       (handle-response http-event)
     [moves http-server-gate]
   ::
+  ++  watch-cache
+    ::
+    ?>  ?=([%clay %writ *] sign)
+    ?~  p.sign  `http-server-gate
+    =/  =beam  [[our r.p.u.p.sign q.p.u.p.sign] q.u.p.sign]
+    =/  url=@t  (spat t.wire)
+    ?.  (~(has by cache.server-state.ax) url)
+      `http-server-gate
+    =/  moves
+      :~
+        [duct %give %cache-changed url beam]
+        [duct %pass wire %c %warp our [q.beam `[%next %x r.beam s.beam]]]
+      ==
+    [moves http-server-gate]
```

Comment:
Just a style nit, but I'd inline the moves variable and just put something like this:
```
:_  http-server-gate
:~  [duct %give ...]
    [duct %pass ...]
==
```